### PR TITLE
Feat[hospitals]: Hospitals page

### PR DESCRIPTION
**Pull request by medimessarat**
# Responsive simple hospitals View
According to 7d33115. I've created a simple responsive view of hospitals with their picture, title and link to Maps data is stored in data.js file. 

Here is a preview of it: 
PR_hospitals

This Fixe the #8,  and branch ready to be merged with master

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    title:`EPH Kouba`,
+    // imageUrl: `https://upload.wikimedia.org/wikipedia/commons/f/ff/Photo_hopital_kouba_26052016.jpg`,
+    imageUrl: `pictures/hospitals/EPH_Kouba.jpg`,
+    location: `36°43'32.7"N 3°05'14.0"E`
+  },
+  {
+    id: uuid(),
+    title:`EPH Bologhine`,
+    imageUrl: `https://cdnfr2.img.sputniknews.com/images//104213/97/1042139753.jpg`,
+    imageUrl: `pictures/hospitals/EPH_Bologhine.jpg`,
```
Comment (mohammedi-haroune) [inline src/data.js:70]: Why there is two `imageUrl` attributes ?

File: package.json [hunk 2]
```diff
@@ -2,10 +2,8 @@
   "name": "red-crescent-app",
   "version": "0.1.0",
   "private": true,
-  "homepage": "http://mohammedi-haroune.github.io/red-crescent-webapp",
   "dependencies": {
     "@material-ui/core": "^4.6.1",
-    "gh-pages": "^2.1.1",
```
Comment (mohammedi-haroune) [inline package.json:8]: why removing gh-pages ?

File: package.json [hunk 3]
```diff
@@ -14,9 +12,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
```
Comment (mohammedi-haroune) [inline package.json:17]: Why removing the deploy scripts ? we need them

File: package.json [hunk 4]
```diff
@@ -14,9 +14,9 @@
...
     "predeploy": "npm run build",
-    "deploy": "gh-pages -d build"
+    "deploy": "gh-pages -d build",
+    "eject": "react-scripts eject",
```
Comment (mohammedi-haroune) [inline package.json]: There is a syntax error here, it should be
```suggestion
    "eject": "react-scripts eject"
```

File: package.json (hunk 2 above)
Comment (medimessarat) [reply package.json:8]: Sorry my mistake, i didn't **git pull** when i created the hospitals-page branch, so i didn't have the version of package.json where it has gh-pages. 
now it's fixed in 098b66e

File: package.json (hunk 3 above)
Comment (medimessarat) [reply package.json:17]: it wasn't remove only the comma was removed, from the version of package.json i had. Now it's fixed 098b66e

**Comment by mohammedi-haroune** [comment]
I've tested it, I really liked it. 

Just one thing: IMHO, clicking on the location button should open a new that rather than use the current tab.

**Comment by mohammedi-haroune** [comment]
A bunch of warning shows up when running:
```
Compiled with warnings.

./src/components/Hospitals/Hospitals.js
  Line 1:17:   'useState' is defined but never used       no-unused-vars
  Line 21:36:  'rest' is assigned a value but never used  no-unused-vars

./src/components/HospitalCard/HospitalCard.js
  Line 1:17:  'useState' is defined but never used     no-unused-vars
  Line 7:3:   'CardActions' is defined but never used  no-unused-vars
  Line 9:3:   'Grid' is defined but never used         no-unused-vars

./src/App.js
  Line 2:8:  'logo' is defined but never used  no-unused-vars
```

File: src/data.js (hunk 1 above)
Comment (medimessarat) [reply src/data.js:70]: There is only one imageUrl attributes, but in two ways: 
1.st way that is **Commented**, where we have the link of the picture.
2.nd  way where the picture are saved and we indicate the relative path.

We'll use the second way, but i kept the first thinking that we might need them until we finally deploy our last version. 
The mistake here is that i forgot to comment all the imageUrl that has link redirecting to the picture. that's why we see two imageUrl, but it's solved now in 1633967

**Comment by mohammedi-haroune** [comment]
The app name and the app icon should also be fixed

**Comment by mohammedi-haroune** [comment]
I have a question, why don't we use the images directly from Wikipedia, this will save us some space and speed up the push/pull process.

**Comment by medimessarat** [comment]
#9 Fixed all reviews, Branch ready to be merged.

**Comment by mohammedi-haroune** [comment]
Icon not yet changed

**Comment by medimessarat** [comment]
I have changed the **react-app icon**, by the **red-crescent icon** with the same name of the file favicon.ico
Here is how it is in the folder:
Capture

**Comment by mohammedi-haroune** [comment]
OK. Sorry

**Pull request merged**